packages/block-editor/src/components/editor-styles/index.js

```diff
@@ -68,7 +69,21 @@ function useDarkThemeBodyClassName( styles ) {
 
 export default function EditorStyles( { styles } ) {
 	const transformedStyles = useMemo(
-		() => transformStyles( styles, EDITOR_STYLES_SELECTOR ),
+		() =>
+			transformStyles(
+				// Assume that non-SVG styles are CSS that can be transformed.
+				styles.filter( ( style ) => style.__unstableType !== 'svgs' ),
+				EDITOR_STYLES_SELECTOR
+			),
+		[ styles ]
+	);
+
+	const transformedSvgs = useMemo(
+		() =>
+			styles
+				.filter( ( style ) => style.__unstableType === 'svgs' )
```

Comment:
Maybe it's worth calling the variable `svgFilter` for clarity?

Reply:
I kept it as just `svgs` because it could be used for things like `mask` and `clip-path` definitions in the future, not just `filter`.

Previously, I've used `svgFilters` in these places, but looking at the design for the [Blue Note](https://github.com/WordPress/community-themes/issues/4) theme, I realized that we might want to support those [other SVG use-cases](https://developer.mozilla.org/en-US/docs/Web/SVG/Applying_SVG_effects_to_HTML_content) as well.